ocs_ci/ocs/pillowfight.py

```diff
@@ -70,51 +69,65 @@ def run_pillowfights(self):
         """
         ocp_local = OCP(namespace=self.namespace)
         pf_files = listdir(constants.TEMPLATE_PILLOWFIGHT_DIR)
-        for pf_yaml in pf_files:
-            pf_fullpath = join(constants.TEMPLATE_PILLOWFIGHT_DIR, pf_yaml)
-            if not pf_fullpath.endswith('.yaml'):
-                continue
-            if not isfile(pf_fullpath):
-                continue
-            pfight = templating.load_yaml(pf_fullpath)
-            lpillowfight = OCS(**pfight)
-            lpillowfight.create()
-            pf_completion_info = ''
-            pf_pod = ''
-            for pillowfight_pod in TimeoutSampler(
-                self.WAIT_FOR_TIME,
-                3,
-                get_pod_name_by_pattern,
-                'pillowfight',
-                self.COUCHBASE_OPERATOR
-            ):
-                try:
-                    pf_pod = pillowfight_pod[0]
-                    pod_info = self.up_check.exec_oc_cmd(
-                        f"get pods {pf_pod} -o json"
-                    )
-                    pf_status = pod_info['status']['containerStatuses'][0]['state']
-                    if 'terminated' in pf_status:
-                        pf_completion_info = pf_status['terminated']['reason']
+        for i in range(replicas):
+            for pf_yaml in pf_files:
+                pf_fullpath = join(constants.TEMPLATE_PILLOWFIGHT_DIR, pf_yaml)
+                if not pf_fullpath.endswith('.yaml'):
+                    continue
+                if not isfile(pf_fullpath):
+                    continue
+
+                # for basic-fillowfight.yaml
+                pfight = templating.load_yaml(pf_fullpath)
+                pfight['metadata']['name'] = 'pillowfight-rbd-simple' + f"{i}"
+
+                lpillowfight = OCS(**pfight)
+                lpillowfight.create()
+
+        pods_info = {}
+        for pillowfight_pod in TimeoutSampler(
+            self.WAIT_FOR_TIME,
```

Comment:
Wait time is set to 600above. 
So, even if n number of pods the runs completes on the all the pods within 600?

Reply:
AFAIK, each pillow-fight client pod hardly takes 5 seconds to complete. With IO config specified in basic-pillowfight.yaml